src/fusiondls/Profile.py

```diff
@@ -373,6 +393,89 @@ def plot_topology(self):
 
         fig.tight_layout()
 
+    def plot(
+        self,
+        mode="Btot",
+        ax=None,
+        legend=False,
+        parallel=True,
+        full_RZ=False,
+        label="",
+        color="teal",
+        **kwargs,
+    ):
+        """
+
+
+        Parameters
+        ----------
+        mode : str
+            What to plot:
+                "Btot" - total B profile
+                "RZ" - RZ space leg profile (excl. above X-point)
+                "Spar_Spol" - Parallel vs poloidal connection length
```

Comment:
I think this will render strangely in the html docs. This would do what you'd expect:

```suggestion
            What to plot:
            - "Btot": total B profile
            - "RZ": RZ space leg profile (excl. above X-point)
            - "Spar_Spol": Parallel vs poloidal connection length
```